lib/wallaby/helpers/key_codes.ex

```diff
@@ -1,8 +1,5 @@
 defmodule Wallaby.Helpers.KeyCodes do
-  @moduledoc """
-  Helper utility for converting key atoms into key codes sutiable to send over
-  the wire.
-  """
+  @moduledoc false
```

Comment:
While we already have the comment, maybe we keep it as a code comment? I'm not a big fan of those, but a high level comment for the module might help for new contributors? Wdyt?